libsql_client/client.py

```diff
@@ -97,4 +101,6 @@ def _normalize_value(in_value: InValue) -> Value:
         return int(in_value.timestamp() * 1000)
     elif isinstance(in_value, bool):
         return int(in_value)
-    return in_value
+    elif isinstance(in_value, (str, float)) or in_value is None:
+        return in_value
+    return bytes(in_value)
```

Comment:
if ok, then @douglasmdev please change this last line to be `bytes(memoryview(in_value))`

Reply:
```py
>>> memoryview(mmap.mmap(-1, 10))
<memory at 0x10a879d80>
>>> bytes(memoryview(mmap.mmap(-1, 10)))
b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
```

Reply:
done!

Reply:
@honzasp `mmap` and others are all covered by `ReadableBuffer`, I'm just defining straight as `bytes` when not typechecking, otherwise it will be undefined and fails.